bunsen/bunsen-core-r4/src/test/java/com/cerner/bunsen/r4/TestData.java

```diff
@@ -0,0 +1,490 @@
+package com.cerner.bunsen.r4;
```

Comment:
Just to be clear: This file is automatically generated and independent of this change; @omarismail94 if you want this to be moved to a separate PR, please let me know.